src/main/resources/scipooldev.properties

```diff
@@ -1,5 +1,7 @@
 SYNAPSE_OAUTH_CLIENT_ID=100055
-TEAM_TO_ROLE_ARN_MAP=[{"teamId":"273957","roleArn":"arn:aws:iam::465877038949:role/ServiceCatalogEndusers"}]
+TEAM_TO_ROLE_ARN_MAP=[{"teamId":"3407239","roleArn":"arn:aws:iam::465877038949:role/ServiceCatalogEndusers"}, //scipool-admin \
```

Comment:
that's a json and json does not support comments and the `\` is not required when splitting string onto multiple lines.